types/wellknown/index.d.ts

```diff
@@ -3,5 +3,8 @@
 // Definitions by: Yair Tawil <https://github.com/yairtawil>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function parse(input: string): {};
+export function parse(input: string): {
```

Comment:
I'm not very familiar with the library.  Doing a quick look in the documentation seems like this method returns a "geometry object" which seem to have different `coordinates` properties based on the type.

Could you confirm this is the correct change? Also as maintainers we appreciate if you could point on the description of the PR the documentation that proves the change.